src/main/resources/avro/biodatamethods.avdl

```diff
@@ -0,0 +1,147 @@
+@namespace("org.ga4gh.methods")
+
+protocol BiodataMethods {
+
+import idl "metadata.avdl";
+import idl "common.avdl";
+import idl "methods.avdl";
+import idl "biodata.avdl";
+
+/******************  /individuals/search  *********************/
+/**
+This request maps to the body of `POST /biosamples/search` as JSON.
+*/
+record SearchIndividualsRequest {
+  /**
+  The dataset to search within.
+  */
+  string datasetId;
```

Comment:
Reference is not ownership.  VCFs in one dataset can reference BAMs in another.  For instances, if you recall 1K genomes variants doesn't mean you put them in the same dataset.

DataSubmission doesn't capture it either because a dataset may not be submitted at one time
(at least there is restriction that it must be).

I think clear terminology is very important.   However, DataSet has been around since the very first version of the API.   So change the term is a bit late.  Instead, improve the documentation.
Lets start a ticket on:
  https://github.com/ga4gh/schemas/blob/master/doc/source/api/apidesign_intro.rst#dataset